### PR TITLE
Fix initial values to multilingual setting variables

### PR DESCRIPTION
**Pull request by nmpetkov**
This make sure variables have initial values when site loads.
Also more robust check in upgrade process.

| Q                 | A
| ----------------- | ---
| Bug fix?          | yes
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Tests pass?       | yes
| Fixed tickets     | - #2330
| Refs tickets      | - #2316
| License           | MIT
| Doc PR            | -
| Changelog updated | no

### CODE REVIEW

**Comment by nmpetkov** [comment]
@Guite there is one syntactic error, please revert to correct.

File: src/lib/util/ModUtil.php [hunk 1]
```diff
@@ -126,6 +126,9 @@ public static function initCoreVars($force=false)
             }
          }
 
+         // Init multilingual variables here, just to have values, even with default site language (page language is set later)
+         self::setupMultilingual()
```
Comment (craigh) [inline src/lib/util/ModUtil.php]: missing `;`

**Comment by craigh** [comment]
>  please revert to correct.

no need to revert. anything other than the missing `;`?

**Comment by craigh** [comment]
corrected missing `;` already

**Comment by craigh** [comment]
https://github.com/zikula/core/commit/9abdff2430ee134ef1e4fcbd5d1d923034c19b32

**Comment by nmpetkov** [comment]
Thank you, something happened with branches and files.
Please test with this changes.

**Pull request merged**